.changeset/cold-cars-relate.md

```diff
@@ -0,0 +1,8 @@
+---
+"@patternfly/pfe-core": minor
+---
+
+`SlotController`: 
+
+- Add `isEmpty` method to check if a slot is empty.  If no slot name is provided it will check the default slot. (#2603)
+- `hasSlotted` method now returns default slot if no slot name is provided. (#2603)
```

Comment:
```suggestion
`SlotController`: Add `isEmpty` method to check if a slot is empty.  If no slot name is provided it will check the default slot. (#2603)
`SlotController`: `hasSlotted` method now returns default slot if no slot name is provided. (#2603)
```